index.html

```diff
@@ -8,13 +8,24 @@
 
   <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
   <link rel="stylesheet" href="css/index.css">
-
 </head>
 
 <body>
     <div class="container">
-   
-        <section class="top-content">
```

Comment:
I had previously stated on the about page that the header/nav area was a great place to use semantic elements.  It looks like you using them, just haven't gotten to putting it into the about page yet.